tools/config/seed.config.ts

```diff
@@ -1,9 +1,15 @@
 import {readFileSync} from 'fs';
 import {argv} from 'yargs';
 import {normalize, join} from 'path';
-import {InjectableDependency} from './seed.config.interface';
+import {InjectableDependency} from './seed.config.interfaces';
 
-const ENVIRONMENTS = {
+interface Environments {
+  DEVELOPMENT: string;
+  PRODUCTION: string;
+  [key: string]: string;
+}
+
+const ENVIRONMENTS: Environments = {
   DEVELOPMENT: 'dev',
   PRODUCTION: 'prod'
 };
```

Comment:
To make `ENVIRONMENTS ` interface "usefull", we may have to move `ENVIRONMENTS ` constant as a class instance member, I suppose. And interface should be moved to `seed.config.interfaces`.